pulp_rpm/app/downloaders.py

```diff
@@ -21,12 +27,15 @@ class RpmDownloader(HttpDownloader):
 
     """
 
-    def __init__(self, *args, silence_errors_for_response_status_codes=set(), sles_auth_token=None,
+    def __init__(self, *args, silence_errors_for_response_status_codes=None, sles_auth_token=None,
```

Comment:
Unrelated change, but using a collection as a keyword argument can lead to problems - best to avoid it.